README.md

```diff
@@ -133,6 +136,11 @@ Must be non-nil and non-empty. The structure of the `pipeline` object is as foll
  containing variables to be interpolated via `{{ }}` in `config_file`.
```

Comment:
oh dang just noticed this is also the old syntax. i'll fix it after merging since this isn't introduced by your change.